reference/customers.v3.yml

```
@@ -1452,7 +1462,7 @@ paths:
             default: application/json
         - name: page
           in: query
-          description: Page number. `page=1`
+          description: Page number. `page` would be ignored when `before` or `after` is provided in the request. `page=1`
```

```suggestion
          description: Page number. `page` will be omitted when `before` or `after` is provided in the request. `page=1`
```
In the Response it will be absent so maybe `omitted` is more appropriate than `ignored`. `ignored` makes sense for the request where if a conflict is present it will be `ignored`

---

This one is actually still the request 😄. The response one is called `pagination`

---

Yeah I lost network access as I was trying to delete this comment 🤣